pages/interface/components/PublishedSince/index.js

```diff
@@ -14,9 +14,9 @@ function formatTooltipLabel(date) {
   return format(new Date(date), "EEEE, d 'de' MMMM 'de' yyyy 'às' HH:mm", { locale: ptBR });
 }
 
-export default function PublishedSince({ date }) {
+export default function PublishedSince({ direction, date }) {
```

Comment:
```suggestion
export default function PublishedSince({ date, ...props }) {
```

Assim o componente fica mais flexível

Reply:
Perfeito.